tools/make_hgrid/create_gnomonic_cubic_grid.c

```diff
@@ -54,7 +54,8 @@
 #include "mosaic_util.h"
 #include "tool_util.h"
 #include "create_hgrid.h"
-
+#define  D2R (M_PI/180.)
```

Comment:
@nikizadehgfdl 
Instead of defining these here, can you #include "constant.h". Otherwise another warning is 
generated with certain compiler flags.

Reply:
@nikizadehgfdl 
More specifically,  can you not define D2R and R2D and just ```#include "constant.h"```

Reply:
@ngs333 thanks. I dropped the two defs and it worked. Pushed.